lib/rules/no-find-dom-node.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
 
         context.report({
           node: callee,
-          message: 'Do not use findDOMNode'
+          message: 'Do not use findDOMNode. It doesn\'t work with function components and is deprecated in StrictMode. See https://reactjs.org/docs/react-dom.html#finddomnode'
```

Comment:
```suggestion
          message: 'Do not use findDOMNode. It doesn‘t work with function components and is deprecated in StrictMode. See https://reactjs.org/docs/react-dom.html#finddomnode'
```
straight quotes are always typographically incorrect in prose :-)

Reply:
I believe you meant to use the right single quotation mark ’ rather than the left one.

Reply:
lol yes, thanks :-)